modules/ROOT/pages/functions/index.adoc

```diff
@@ -797,6 +798,18 @@ Database functions provide information about databases.
 label:new[Introduced in 5.12]
 |===
 
+[[header-query-functions-genai]]
+**xref::genai-integrations.adoc#single-embedding[GenAI function]** label:new[Introduced in 5.16] label:aura-only[Available only on Aura]
+
+This function is used to generate a vector embedding for a single value.
```

Comment:
Should there be a link to the plugin docs?

Reply:
This link is to the plugin docs.

Reply:
I mean `xref::genai-integrations.adoc#single-embedding`

Reply:
Oh, sorry, I'm just blind!